### PR TITLE
OTKA-751031 build-sso-integration > saml2 Acrolinx improvements

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Some words, phrases... for Acrolinx
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-751031](https://oktainc.atlassian.net/browse/OKTA-751031)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/prep.md](https://github.com/okta/okta-developer-docs/blob/1e1db893fd46ee257ece7f46b4edffcb651aec35/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-sso-integration%2Fmain%2Fsaml2%2Fprep.md) | [76](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e7e253a8-c1b6-4bfe-81b9-7d98b966ed6b) | 17  | 64  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/see-also.md](https://github.com/okta/okta-developer-docs/blob/1e1db893fd46ee257ece7f46b4edffcb651aec35/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-sso-integration%2Fmain%2Fsaml2%2Fsee-also.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bb6edcd6-627c-419e-9e54-1ea98278612f) | 5  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.53987d3c-8a0b-43ed-87d0-ff4208faffee?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/prep.md](https://github.com/okta/okta-developer-docs/blob/6d894786944729d11d0b4fa2a2de06f7b019e762/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-sso-integration%2Fmain%2Fsaml2%2Fprep.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ddd88a6d-863a-4995-827c-45a15250997c) | 17  | 64  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/see-also.md](https://github.com/okta/okta-developer-docs/blob/6d894786944729d11d0b4fa2a2de06f7b019e762/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-sso-integration%2Fmain%2Fsaml2%2Fsee-also.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2481b9e1-6808-4db5-83d9-5f6f50a0e4f8) | 5  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.edf2a55c-06d4-43f9-9023-1263038cd34a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/prep.md [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
 
 In a SAML integration, Okta is the Identity Provider (IdP), and your app is the Service Provider (SP).
 
-1. Determine the default Assertion Consumer Service (ACS) URL for your integration. This is often referred to as the SP sign-in URL. This is the endpoint on your application where the SAML responses are posted.
-1. Find your Audience URI. This is sometimes referred to as the SP entity ID or the entity ID of your app.
-1. (Optional) Set up a Default Relay State page, where users land after they successfully sign in to the SP using SAML. This must be a valid URL.
-1. Gather any required SAML attributes. You can choose to share Okta user profile values as SAML attributes with your app.
+1. Determine the default assertion consumer service (ACS) URL for your integration. This is often referred to as the SP sign-in URL. This is the endpoint on your app where the SAML responses are posted.
+1. Find your audience URI. This is sometimes referred to as the SP entity ID or the entity ID of your app.
+1. Set up a default relay state page, where users land after they successfully sign in to the SP using SAML. This must be a valid URL. (Optional)
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/prep.md:27]:
```suggestion
1. Optional. Set up a default relay state page, where users land after they successfully sign in to the SP using SAML. This must be a valid URL. 
```

Comment (grahamsmith-okta) [reply packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/prep.md:27]: On the prod side, we recommend putting `Optional.` at the beginning of the step, no parentheses. Not sure if the dev docs recommendation is different?

**Review by grahamsmith-okta — APPROVED** [review]
one comment

File: packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/prep.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/prep.md:27]: Acrolinx doesn't like it if the list isn't consistent. For this numbered list, Acrolinx is looking for an action verb at the beginning of every item. I can try to use `Optional.` and check the score again. I feel pretty bad that we have to lowercased ".NET framework" for the score to get over 80 :-(

Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/prep.md:27]: Just checked. I'm under 80 if I use `Optional.` at the beginning. Here's the Acrolinx message:
image

**Pull request merged**